script/test/utils.sh

```diff
@@ -63,6 +63,12 @@ CONTAINERD_FLAGS+="--address ${CONTAINERD_SOCK#"unix://"} \
 
 containerd_groupid=
```

Comment:
This is set on L88 using `ps -o ... `. That doesn't work on windows. We should set containerd_groupid=pid for windows only.

Reply:
Done